### PR TITLE
[risk=no] rm genomics java build

### PR DESCRIPTION
**Pull request by calbach**
We used this early on to generate some synthetic data. We're no longer expecting to use this in the near term, and it's a relatively heavy dependency and it's currently broken. Remove for now - this PR can always be reverted if we decide we need this functionality in the future.

### CODE REVIEW

**Review by jmthibault79 — COMMENTED** [review]
Changes look good.  App runs fine without this.  But we can remove more.

Important to remove in this PR:
`randomize-vcf` in `devstart.rb`
genomics/wdl/RandomizeVcf

Optional/future:
genomics/docs
genomics/wdl
genomics/scripts
genomics/README.md

File: api/libproject/devstart.rb [hunk 1]
```diff
@@ -585,7 +585,7 @@ def create_cdr_indices(cmd_name, *args)
   content_type = "Content-Type: application/json"
   accept = "Accept: application/json"
   circle_token = "Circle-Token: "
-  payload = "{ \"branch\": \"#{op.opts.branch}\", \"parameters\": { \"wb_create_cdr_indices\": true, \"cdr_source_project\": \"#{cdr_source}\", \"cdr_source_dataset\": \"#{op.opts.bq_dataset}\", \"wgv_source_project\": \"#{op.opts.wgv_project}\", \"wgv_source_dataset\": \"#{op.opts.wgv_dataset}\", \"wgv_source_table\": \"#{op.opts.wgv_table}\", \"project\": \"#{op.opts.project}\", \"cdr_version_db_name\": \"#{op.opts.cdr_version}\", \"array_source_table\": \"#{op.opts.array_table}\", \"data_browser\": #{op.opts.data_browser} }}"
+  payload = "{ \"branch\": \"#{op.opts.branch}\", \"parameters\": { \"wb_create_cdr_indices\": true, \"cdr_source_project\": \"#{cdr_source}\", \"cdr_source_dataset\": \"#{op.opts.bq_dataset}\", \"wgv_source_project\": \"#{op.opts.wgv_project}\", \"wgv_source_dataset\": \"#{op.opts.wgv_dataset}\", \"wgv_source_table\": \"#{op.opts.wgv_table}\", \"project\": \"#{op.opts.project}\", \"cdr_version_db_name\": \"#{op.opts.cdr_version}\", \"data_browser\": #{op.opts.data_browser}\", \"array_source_table\": #{op.opts.array_table} }}"
```
Comment (jmthibault79) [inline api/libproject/devstart.rb:588]: is this change correct?

Comment (calbach) [reply api/libproject/devstart.rb]: Nope, thanks - reverted

**Comment by calbach** [comment]
Good catch, removed devstart cmd / readme, and some of the relevant WDLs.

- `docs/` is pretty much all still relevant 
- some of the WDLs **may** still be relevant, but I'd need to pick through these
- scripts are in the same boat

I can take another look at those later when sorting through the workflow details.

**Review by jmthibault79 — APPROVED** [review]

**Pull request merged**